portfolio/src/main/webapp/script.js

```diff
@@ -226,3 +226,43 @@ function requestTranslation() {
   const languageQuery = updateQueryString('language', selectedLanguage);
   refreshComments(languageQuery);
 }
+
+/** Creates a map and adds it to the page. */
+function initMap() {
+  // Try HTML5 geolocation.
+  var map = new google.maps.Map(
+      document.getElementById('map'), {center: {lat: 42.280827, lng: -83.743034}, zoom: 5});
+  var infoWindow = new google.maps.InfoWindow;
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(function(position) {
```

Comment:
Here and below: is there a reason you're using the `function` syntax over the arrow function syntax?

Reply:
Because I got the code from https://developers.google.com/maps/documentation/javascript/geolocation and that was the style they used and it worked, but you are right, an arrow function syntax may be more readable